src/components/atoms/empty-state.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import Icon, { StyledIcon } from './icon'
+import { colors } from '../../tokens'
+
+const Wrapper = styled.div`
+  width: 100%;
+  height: 100%;
+  text-align: center;
+  ${StyledIcon} {
+    line-height: 1em;
+  }
+`
+
+const Content = styled.div`
+  margin-top: 1em;
```

Comment:
Whoops, sorry I didn't notice this earlier. `em` is relative, we're trying to lock the values in by using [_spacing tokens_](https://github.com/auth0/cosmos/blob/master/src/tokens/spacing.js)

Reply:
Whoops! I'll fix that.

Reply:
Added a new commit that fixes this. I left the `line-height: 1em` for the icon image, since the purpose there is really just to treat the icon as an image rather than an in-line glyph.